modules/tinymce/src/core/main/ts/api/file/BlobCache.ts

```diff
@@ -90,30 +91,25 @@ export const BlobCache = function (): BlobCache {
     };
   };
 
-  const add = function (blobInfo: BlobInfo) {
+  const add = (blobInfo: BlobInfo) => {
     if (!get(blobInfo.id())) {
       cache.push(blobInfo);
     }
   };
 
-  const get = function (id: string): BlobInfo {
-    return findFirst(function (cachedBlobInfo) {
-      return cachedBlobInfo.id() === id;
-    });
-  };
+  const findFirst = (predicate: (blobInfo: BlobInfo) => boolean) => Arr.find(cache, predicate).getOrUndefined();
 
-  const findFirst = function (predicate: (blobInfo: BlobInfo) => boolean) {
-    return Arr.filter(cache, predicate)[0];
-  };
+  const get = (id: string): BlobInfo =>
```

Comment:
Should `undefined` be added as return type here to match with the interface?

Reply:
Yes it should, thanks for the catch :+1:

Reply:
I've just removed the return types here as they really aren't needed given we have the interface, so they end up just being duplicate code.